### PR TITLE
[EXTRA] fix function call & show modal

### PR DESCRIPTION
**Pull request by mishaadzh**
Call order functions:

1.  no active rent - delist & withdraw , handeDelist()
2.  active rent - delist, handleDelist()
3. delisted & actie - button disabled
4. delisted & no active rent - withdraw, handleWithDraw()

Open modal when you have a listed property which you have delisted but there is no active rent

### CODE REVIEW

File: src/modules/land-works/views/single-land-view/index.tsx [hunk 1]
```diff
@@ -135,12 +136,16 @@ const SingleLandView: React.FC = () => {
     }
   };
 
-  const handleDelistButton = () => {
+  const handleContructFunctions = () => {
```
Comment (failfmi) [inline src/modules/land-works/views/single-land-view/index.tsx:139]: `handleContructFunctions` -> `handleConstructFunctions`?

File: src/modules/land-works/views/single-land-view/index.tsx [hunk 2]
```diff
@@ -120,6 +120,7 @@ const SingleLandView: React.FC = () => {
   };
 
   const handleWithdraw = async () => {
+    console.log('withdraw');
```
Comment (failfmi) [inline src/modules/land-works/views/single-land-view/index.tsx:123]: Could we remove this log

File: src/modules/land-works/views/single-land-view/index.tsx [hunk 3]
```diff
@@ -215,15 +220,19 @@ const SingleLandView: React.FC = () => {
     setClaimButtonDisabled(!asset?.unclaimedRentFee?.gt(0));
   }, [asset]);
 
+  const isWithDraw = () => {
```
Comment (failfmi) [inline src/modules/land-works/views/single-land-view/index.tsx:223]: `isWithdraw`

File: src/modules/land-works/views/single-land-view/index.tsx (hunk 1 above)
Comment (mishaadzh) [reply src/modules/land-works/views/single-land-view/index.tsx:139]: handleContractFunctions

**Review by failfmi — APPROVED** [review]

**Pull request merged**